### DISCUSSION TITLE
milvus 2.2.8 Business feedback queries in use are slow

### DISCUSSION THREAD

**Discussion by 1292253144** [milvus-io/milvus#27860] [Q&A and General discussion]
集合数据量82W+，查询比较慢，会是什么原因，如果排查
![image](https://github.com/milvus-io/milvus/assets/123862896/e4dacd3c-c9ce-40bf-bda8-437ed5aecf6b)

**Comment by yhmo** [comment]
数据是固定的？有没有增删操作？

**Comment by 1292253144** [comment]
如果有数据变更避免flush操作；如果没有数据变更呢？数据变更是针对集合级别吗？
